waiter/integration/waiter/basic_test.clj

```diff
@@ -32,6 +32,28 @@
             [waiter.util.utils :as utils])
   (:import (java.net URLEncoder)))
 
+(defn- retrieve-expected-resource-usage
+  [waiter-url service-id {:keys [cpus mem]}]
```

Comment:
Doesn't service-description use `:strs` (not `:keys`)?

Reply:
Unfortunately the helper function keywordizes the maps as part of processing the API response that gets us this data :(